pkg/agent/baker.go

```diff
@@ -957,6 +957,9 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"ShouldEnableCustomData": func() bool {
 			return !config.DisableCustomData
 		},
+		"GetArtifactCacheContainerRegistryServer": func() string {
```

Comment:
where is this used ?

Reply:
used for NetworkIsolatedCluster. The private acr to pull OCI artifact (mar artifact cached in acr via artifact cache)

Reply:
will it be better to parse this variable from agent baker data model manageCluster?